### DISCUSSION TITLE
3.0.0 rollup build failure since esm upgrade under node 12

### DISCUSSION THREAD

**Discussion by sync** [Swatinem/rollup-plugin-dts#140] [Q&A] [answered]
I am having issues using this plugin since conversion to native ES modules.

Rollup failure points to:
https://rollupjs.org/guide/en/#using-untranspiled-config-files

but resolution seems to be for Node 13+ only.

this is the error

```
[!] Error: While loading the Rollup configuration from "rollup.config.js", Node tried to require an ES module from a CommonJS file, which is not supported. A common cause is if there is a package.json file with "type": "module" in the same folder. You can try to fix this by changing the extension of your configuration file to ".cjs" or ".mjs" depending on the content, which will prevent Rollup from trying to preprocess the file but rather hand it to Node directly.
```

**Comment by Swatinem** [comment, marked as answer]
Meh :-(

I released a fixed `3.0.1`.

BTW, first time I see a github "Discussions" being used. I’m still a bit confused by this :-D An issue would have been better in this case.

**Comment by sync** [comment]
Agreed, an issue would have been nicer and thanks for the fix I’ll get my ci to pull latest version.

**Reply by sync** [reply]
Working now thanks once more
